envs/vpn/conda/fedbiomed-node.yaml

```diff
@@ -10,56 +10,55 @@ channels:
 
 dependencies:
   # minimal environment
-  - python >=3.9,<3.10
-  - pip
-  - jupyter
-  - ipython
+  # python 3.11 recently released 2022-11 and not yet supported by some deps including: torchvision
+  - python >=3.10,<3.11
+  - pip >= 23.0
+  - jupyter ~=1.0.0
+  - ipython ~=8.13.2
   # tests
-  - pytest >6.2.2
-  - tinydb >=4.4.0,<5.0.0
-  - tabulate >=0.8.9,<0.9.0
+  - pytest ~=7.2.0
+  - pytest-cov ~=4.1.0
+  - tinydb ~=4.7.1
+  - tabulate >=0.9.0,<0.10.0
   # code
-  - GitPython >=3.1.14,<4.0.0
-  - requests >=2.25.1,<3.0.0
-  - paho-mqtt >=1.5.1,<2.0.0
-  - validators >=0.18.2,<0.19.0
-  - tqdm >=4.59.0,<5.0.0
-  - git
-  - packaging >=23.0,<24.0
+  - requests ~=2.29.0
+  - paho-mqtt ~=1.6.1
+  - validators >=0.20.0,<0.21.0
+  - tqdm ~=4.65.0
```

Comment:
I didn't see this module is used somewhere in Fed-BioMed. May it left from an old implementation?

Reply:
`tqdm` is used in `download_and_split_ixi.py` script, which is used in medical folder dataset notebook.

As discussed, we may remove from the `yml` file and replace with instructions for `pip install tqdm` in the notebooks + documentation. Doing this.